gpii/node_modules/packagekit/nodepackagekit/nodepackagekit_test.js

```diff
@@ -155,6 +155,17 @@ var fluid = require("universal"),
         return matchPkg;
     };
 
+    pkTests.testPerformActionFailure = function() {
```

Comment:
Thanks for the link, @the-t-in-rtf !
Aside:  as a test, I added the hook to my copy of universal, added a lint violation to gpii.js, and tried to commit it.  Nothing happened.  It turns out that universal's Gruntfile.js checks only the files under the "`gpii`", "`tests`", and "`examples`" directories.  Adding "`./*.js`" to the list fixed that and reported two errors in `gpii.js`, besides my cooked-up one.  Is there a reason for not linting the files in the top level directory?